src/components/common/button/button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button as ReactButton, View } from 'react-native';
+
+type Props = {
+    label: string;
+    onClick?: (evt: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+const Button: React.FC<Props> = ({
+    label,
+    onClick,
+}) => {
+    return (
+        <View>
+            <ReactButton title={label} onPress={onClick} color={'#f3427f'} />
```

Comment:
Color should be populated from predefined values and should be selected by incoming prop param